htmresearch/regions/ApicalTMRegion.py

```diff
@@ -287,6 +295,7 @@ def __init__(self,
                initialPermanence=0.21,
                connectedPermanence=0.50,
                minThreshold=10,
+               reducedThresholdBasal=11, # Only used for apicalTiebreak implementation
```

Comment:
We should use the same default value in the `ApicalTMRegion` as in the `ApicalTiebreakTemporalMemory`.